docs/source/_static/rest-api.yml

```diff
@@ -135,6 +135,27 @@ paths:
           schema:
             type: boolean
           allowEmptyValue: true
+        - name: sort
+          in: query
+          description: |
+            Sort users by this key (new in JupyterHub 5.0).
+            Default: id (opaque but stable internal order).
+          schema:
+            type: string
+            enum:
+              - id
+              - name
+              - last_activity
```

Comment:
I think this needs an update as well, so that it mentions the `-` prefix. Also, this schema won't make a request fail if its passes `-id` instead of the listed enums that only includes `id` right?

Reply:
right, forgot to push. I debated that. I don't think adding `-` to enum looks or feels nice, so I removed the enum and added examples, instead. This is what it looks like after #4700:

<img width="917" alt="Screenshot 2024-03-06 at 13 26 43" src="https://github.com/jupyterhub/jupyterhub/assets/151929/bc2b2dda-51fa-4cd9-b8c1-380f3e4efd49">

Reply:
> Also, this schema won't make a request fail if its passes -id instead of the listed enums that only includes id right?

Right, should clarify: this schema exists solely for the purposes of generating documentation for consumption by _humans_. It has no effect on JupyterHub behavior, and is not intended or appropriate for code generation or validation purposes. We exclusively use OpenAPI as a _rendering medium_ for documentation, not as a strict specification of our API. Deviations from the spec where they produce nicer-looking or clearer docs are preferred to strict adherence via complex things like `oneOf`, etc.